### DISCUSSION TITLE
uso de || en los ejercicios de practica

### DISCUSSION THREAD

**Discussion by rinrot** [INGCOM-UNRN-P1/cursada-2023#48] [Preguntas y respuestas] [answered]
Hola! estaba haciendo la correccion del ejercicio 6 del tp1 y viendo videos sobre los condicionales me encontre con el uso de "||" como un or. lo podemos usar para realizar los ejercicios?

**Comment by martinvilu** [comment, marked as answer]
Seeee


Compartí también!

**Comment by rinrot** [comment]
se ve _bastante_ feo pero funciona joya!

`if ((dividendo < 0 && divisor > 0) || (dividendo > 0 && divisor < 0))`
